src/client/actions/MessageReactionRemove.js

```diff
@@ -31,6 +31,7 @@ class MessageReactionRemove extends Action {
     reaction._remove(user);
     /**
      * Emitted whenever a reaction is removed from a cached message.
+     * Requires the GUILD_MESSAGE_REACTIONS or DIRECT_MESSAGE_REACTIONS intent.
```

Comment:
Might need to specify which is for what (so people don't expect DIRECT_MESSAGE_* to return things from guilds)

Reply:
isn't that kinda common sense though

Reply:
You'd be surprised... 😓